### PR TITLE
Add version to build and the archived asar app

### LINKED ISSUE #853: Generate and publish `version.json` and `excalidraw-#####.zip` on build
Related: https://github.com/excalidraw/excalidraw-desktop/issues/23

### PR DESCRIPTION
**Pull request by lipis**
- [x] Generate [`version.json`](https://excalidraw-git-version.excalidraw-team.now.sh/version.json) (The actual filename can be found there)
- [x] Zip the build folder with a name `build/excalidraw-2020-02-28-21-42-03.asar`

------

- https://excalidraw-git-version.excalidraw-team.now.sh/version.json
- https://excalidraw-git-version.excalidraw-team.now.sh/excalidraw-2020-02-28-21-42-03.asar

Fixes #853

### CODE REVIEW

File: scripts/build-version.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  )}`;
+  return `${date_}-${time}`;
+};
+
+const now = new Date();
+
+const data = JSON.stringify(
+  {
+    app: `excalidraw-${versionDate(now)}.zip`,
+    version: versionDate(now),
```
Comment (j-f1) [inline scripts/build-version.js]:
```suggestion
    version: now.toISOString().split('T')[0],
```

Comment (j-f1) [reply scripts/build-version.js]: Also, does this mean there will only be one update per day?

Comment (lipis) [reply scripts/build-version.js]: that's why I included hour/min/sec

Comment (j-f1) [reply scripts/build-version.js]: Ah — why not use the iso8601 format?

**Comment by dwelle** [comment]
Few things:

1. the tasks aren't Windows-compatible since it appears the file shebang isn't enough. I'd prepend `node ` to the scripts to solve it.
2. is the `jszip-cli` meant to add itself to env variables? On Windows it doesn't seem to do so, as I can't use it from command line.

If we want to eschew cross-platform support, I guess we can ignore these, but I won't be able to review or debug future issues if they come up.

**Comment by lipis** [comment]
I updated using `asar` something that Electron understands..

**Comment by lipis** [comment]
Could you test again?

**Comment by dwelle** [comment]
Works! I tweaked the scripts.

**Comment by lipis** [comment]
Yes, I'm aware of it.. but it doesn't matter in real life.

**Pull request merged**